### PR TITLE
Add terraform plan for Azure Container Instances

### PR DESCRIPTION
**Pull request by 5minpause**
I created this plan for deploying an image as container instance. For sharing it here I variable-ized the hell out of it.
There are things like the ENV Vars that are not yet possible to set as variables, but I guess that will be done by the user of the script.

LMK what you think or if you need changes.
(Beware, I am on vacation soon until start of August…)

### CODE REVIEW

**Comment by selamanse** [comment]
Hi @5minpause, first: thanks for your contribution and sorry for delayed review.

just a few requests before we can merge this:

File: terraform-azure-container-instances/main.tf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+terraform {
+}
+
+provider "azurerm" {
```
Comment (selamanse) [inline terraform-azure-container-instances/main.tf:4]: please remove this provider assignment as we do not declare those directly in modules for reasons that are described best here: https://www.terraform.io/docs/language/modules/develop/providers.html#legacy-shared-modules-with-provider-configurations

**Review by selamanse — COMMENTED** [review]
please also take care about following 2 things:

- format files accordingly with `terraform fmt`
- configure a workflow for this module by using [an existing workflow definition](blob/main/.github/workflows/terraform-azure-sftp.yml) as a basis so it can pass a basic quality gate.

**Review by selamanse — CHANGES_REQUESTED** [review]
see comments above

**Comment by 5minpause** [comment]
Thanks for your feedback. Sorry for my late reply. Since I left EDDI I will not be able to work on this project any longer.
Take care. Bye 👋

**Pull request closed without merging**